docs/user/convert/parquet.rst

```diff
@@ -0,0 +1,171 @@
+.. _parquet_converter:
+
+Parquet Converter
+=================
+
+The Parquet converter handles data written by `Apache Parque <https://parquet.apache.org/>`__. To use the Parquet
+converter, specify ``type = "parquet"`` in your converter definition.
+
+Configuration
+-------------
+
+The Parquet converter supports parsing whole Parquet files. Due to the Parquet random-access API, the file path
+must be specified in the ``EvaluationContext``. Further, pure streaming conversion is not possible (i.e. using
+bash pipe redirection into the ``ingest`` or ``convert`` command).
+
+As Parquet does not define any object model, standard practice is to parse a Parquet file into Avro GenericRecords.
+
+The Avro GenericRecord being parsed is available to field transforms as ``$0``.
+
+Avro Paths
+----------
+
+Because Parquet files are converted into Avro records, it is possible to use Avro paths to select elements. See
+:ref:`avro_converter` for details on Avro paths. Note that the result of an Avro path expression will be typed
+appropriately according to the Parquet column type (e.g. String, Double, List, etc).
+
+.. _parquet_converter_functions:
+
+Parquet Transform Functions
+---------------------------
+
+GeoMesa defines several Parquet-specific transform functions, in addition to the ones defined under
+:ref:`avro_converter_functions`.
+
+parquetPoint
```

Comment:
When are these needed?  The example below uses paths and existing functions.  Is this syntactical sugar?

Reply:
These are for geomesa-encoded geometry columns, as written by geomesa-fs-storage-parquet. The examples below are just pulling lat/lon out of generic parquet data, so they're not needed. Our geometry columns use more complicated nested structures, that wouldn't be easy (possible?) to read using regular paths and functions.

Reply:
generally these will probably only show up in type-inferenced ingestion of geomesa-parquet files.